concent_api/core/views.py

```diff
@@ -93,6 +94,14 @@ def receive(request, _message):
                     decoded_message_data,
                     last_delivered_message_status,
                 )
+        subtask_results_settled_timing = settings.SUBTASK_VERIFICATION_TIME + settings.FORCE_ACCEPTANCE_TIME + settings.CONCENT_MESSAGING_TIME
+        if (
+            last_delivered_message_status.message.type                           == message.concents.ForceSubtaskResults.TYPE and
+            last_delivered_message_status.message.auth.provider_public_key_bytes == client_public_key and
+            current_time > decoded_message_data.ack_report_computed_task.timestamp + subtask_results_settled_timing
```

Comment:
I think I have written somewhere before that we should look at the ack timestamp but after going through the docs recently I see that this should be the timestamp from `ReportComputedTask`. If there are more places to change than just this one, you can submit a separate ticket and leave this here as is.

Reply:
Yep, it's used in couple places, just as #41 suggested, so I'm not changing it right now.